testsuite/tests/regression/pr11865/mkroot_stubs.c

```diff
@@ -0,0 +1,50 @@
+#include <stdlib.h>
+#include <caml/custom.h>
+#include <caml/memory.h>
+#include <caml/fail.h>
+
+struct box {
+  value nongen_root;
+  value gen_root;
+};
+
+static void box_finalize(value v)
+{
+  struct box** r = Data_custom_val(v);
+  caml_remove_global_root(&(*r)->nongen_root);
+  caml_remove_generational_global_root(&(*r)->gen_root);
+  free(*r);
+}
+
+static struct custom_operations box_ops = {
+  "_box",
```

Comment:
Identifiers starting with "_" are (informally) reserved to the OCaml runtime.
```suggestion
  "box",
```

Reply:
Fixed